test/js_optimizer/minimal-runtime-2-emitDCEGraph.js

```diff
@@ -65,11 +65,14 @@ var imports = {
  }
 };
 var _main, _unused, wasmExports;
-WebAssembly.instantiate(Module["wasm"], imports).then(((output) => {
- wasmExports = output.instance.exports;
+function assignWasmExports(wasmExports) {
  _main = wasmExports["b"];
  _unused = wasmExports["c"];
- initRuntime();
+}
+WebAssembly.instantiate(Module["wasm"], imports).then(((output) => {
+ wasmExports = output.instance.exports;
+ assignWasmExports(wasmExports);
+ initRuntime(wasmExports);
  ready();
 }));
```

Comment:
This looks a little larger, but I guess it gets optimized by closure?

Reply:
Yes, closure just inlines this.

Reply:
As you can see its a code size win actually

Reply:
Wait.. sorry, this change is code size neutral.. I was getting confused between changes.